extensions/amp-analytics/0.1/events.js

```diff
@@ -22,6 +22,30 @@ import {user} from '../../../src/log';
 const VARIABLE_DATA_ATTRIBUTE_KEY = /^vars(.+)/;
 const NO_UNLISTEN = function() {};
 
+/** @enum {string} */
+const WAITFOR_EVENTS_LIST = {
+  NONE: 'none',
+  INI_LOAD: 'ini-load',
+  RENDER_START: 'render-start',
+};
+
+/**
+ * @interface
+ */
+class SignalTrackerDef {
+  /**
+   * @param {?string} unusedEventType
```

Comment:
The `IniLoadTracker` simply doesn't need to use it. But the argument itself is still required. Basically, the caller is required to supply it according to the interface. The implementation class doesn't have to use if it already knows it. If you'd like, you can do `dev().assert(eventType == 'ini-load')` - that would make this argument "used".